packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -221,7 +252,7 @@ export class ChainService implements ChainServiceInterface {
 
   async concludeAndWithdraw(
     finalizationProof: SignedState[]
-  ): Promise<providers.TransactionResponse | void> {
+  ): Promise<providers.TransactionResponse> {
```

Comment:
Returning `void` or a `TransactionResponse` seemed like a code smell and made typing harder.